### PR TITLE
AWS AMI ID, package names, and permission changes

### PR DESCRIPTION
**Pull request by ckochenower**
Package name and dependency hierarchy changes since moving codebase from
evernym/sovrin-environments to hyperledger/indy-node/envrionment
required changes and improvements.

Signed-off-by: Corin Kochenower <corin.kochenower@evernym.com>

### CODE REVIEW

**Comment by hyperledger-github** [comment]
Could one of the committers please verify this patch?

File: environment/cloudformation/training/IndyCluster.json [hunk 1]
```diff
@@ -1393,7 +1393,7 @@
                     {
                       "Ref": "Validator01OperatingSystem"
                     },
-                    "indy"
+                    "sovrin"
```
Comment (ashcherbakov) [inline environment/cloudformation/training/IndyCluster.json]: This is Indy repository. It should have no Sovrin dependencies.

Comment (ckochenower) [reply environment/cloudformation/training/IndyCluster.json]: @ashcherbakov - This is just a key/name used to lookup a value in the RepositoryKeys hash here using Fn::FindInMap:

https://github.com/hyperledger/indy-node/blob/master/environment/cloudformation/training/IndyCluster.json#L5067

When you say this CloudFormation should not have Sovrin dependencies, are you referring to this repo URI?

https://github.com/hyperledger/indy-node/blob/master/environment/cloudformation/training/IndyCluster.json#L5078-L5079

A more detailed explanation is as follows:
The "68DB5E88" is appended to the following apt-key command when Fn::FindInMap is called on the RepositoryKeys map with the name of the operating system and the name of the repo:
`                "apt-key adv --keyserver keyserver.ubuntu.com --recv-keys ",
                {
                  "Fn::FindInMap": [
                    "RepositoryKeys",
                    {
                      "Ref": "Validator01OperatingSystem"
                    },
                    "indy"
                  ]
                },
                "\n",
`
The following add-apt-repository command adds debian repo "https://repo.sovrin.org/deb xenial" based on operating system and branch/codebase

`
                "add-apt-repository \"",
                {
                  "Fn::FindInMap": [
                    "RepositoryEvernym",
                    {
                      "Ref": "Validator01OperatingSystem"
                    },
                    {
                      "Ref": "CodeBase"
                    }
                  ]
                },`

**Comment by ckochenower** [comment]
@ashcherbakov - I changed "sovrin" to "indy" in the cloudformation map names and key names within those maps. The repository is still repo.sovrin.org here:

https://github.com/hyperledger/indy-node/blob/master/environment/cloudformation/training/IndyCluster.json#L5078-L5079

If this is not correct, please advise.

File: environment/cloudformation/training/IndyCluster.json [hunk 2]
```diff
@@ -1425,10 +1425,14 @@
                 "\"\n",
                 "apt-get update\n",
                 "DEBIAN_FRONTEND=noninteractive apt-get install -y debsigs debsig-verify apt-transport-https\n",
-                "DEBIAN_FRONTEND=noninteractive apt-get install -y dialog figlet python-pip python3-pip python3.5-dev libsodium18 unzip make screen indy-node tmux vim wget\n",
+                "DEBIAN_FRONTEND=noninteractive apt-get install -y dialog figlet python-pip python3-pip python3.5-dev unzip make screen sovrin tmux vim wget\n",
```
Comment (ashcherbakov) [inline environment/cloudformation/training/IndyCluster.json]: It must install `indy-node`, not `sovrin`. `indy-node` contains all necessary dependencies and scripts.

**Pull request merged**